docs/eog/case-studies.rst

```diff
@@ -0,0 +1,12 @@
+.. _Case-Studies:
+
+Case Studies
+=============
+
+
+
+.. toctree::
+   :maxdepth: 1
+   :caption: Sections:
+
+   case-studies-cw20
```

Comment:
I think it is worth adding authorship to each case study - that person (or persons) can go on the main author list also - but I think given the somewhat standalone nature of the case study - I think that would be ok - what do you think @rainsworth ?

Reply:
Yes, that makes sense. I can't find in the restructured text primer how to e.g. add an author to this TOC where you've made the comment, so I've added authorship to the first page of the cw20 case study.

Reply:
That's fine